### PR TITLE
Features/update sdk ios 6.3.2

### PR DESCRIPTION
**Pull request by khmMouna**

### CODE REVIEW

**Comment by khmMouna** [comment]
deleteState est setSate avec value = nil, comme dans le native sdk iOS et sur cordova

**Comment by oristanovic** [comment]
Ok but then it won't work:

 if (!name || !value) {
        return;
  }

**Comment by khmMouna** [comment]
- [ ]  Add custom event

- [ ]  Add deleteState

- [ ]  Remove "created by...", use the same as in older classes

File: ios/RNAcc/RNAccTracking.m [hunk 1]
```diff
@@ -94,5 +94,39 @@ - (dispatch_queue_t)methodQueue {
...
+        }
+    }];
+    
+    [Accengage trackEvent:eventType withCustomParameters:customEventParams];
+}
+
+
+#pragma mark - Helper Methods
+
+- (NSDate*)dateFromString:(NSString*)stringParam {
```
Comment (oristanovic) [inline ios/RNAcc/RNAccTracking.m]: The best would be to move this method to a separate class

**Pull request merged**